go/worker/compute/merge/committee/node.go

```diff
@@ -273,8 +274,21 @@ func (n *Node) HandleNewBlockLocked(blk *block.Block) {
 // HandleResultsFromExecutorWorkerLocked processes results from an executor worker.
 // Guarded by n.commonNode.CrossNode.
 func (n *Node) HandleResultsFromExecutorWorkerLocked(spanCtx opentracing.SpanContext, commit *commitment.ExecutorCommitment) {
-	// TODO: Context.
-	if err := n.handleResultsLocked(context.TODO(), commit); err != nil {
+	ctx, err := n.commonNode.Group.RoundContext()
```

Comment:
Actually I just noticed that there already eixsts a `n.roundCtx` that you can use, so this is not needed.

Reply:
Haha, this PR is turning into a 2-line fix :D

Reply:
:-)